gms_app/models.py

```diff
@@ -2,36 +2,84 @@
 
 
 class Garden(models.Model):
-    name = models.CharField(max_length=50)
-    description = models.TextField(blank=True)
+    name = models.CharField(max_length=50, verbose_name="Garden Name", help_text="Enter the name of the garden")
+    description = models.TextField(blank=True, help_text="Enter a brief description of the garden")
 
     def __str__(self):
         return self.name
 
+    class Meta:
+        verbose_name = "Garden"
+        verbose_name_plural = "Gardens"
+        ordering = ["name"]
+
 
 class GardenBed(models.Model):
-    garden = models.ForeignKey(Garden, blank=True, on_delete=models.DO_NOTHING, null=True)
-    name = models.CharField(max_length=100)
-    description = models.TextField(blank=True)
-    location = models.JSONField(default=dict)
+    garden = models.ForeignKey(Garden, blank=True, null=True, on_delete=models.SET_NULL, related_name="beds", help_text="Select the garden this bed belongs to")
+    name = models.CharField(max_length=100, verbose_name="Bed Name", help_text="Enter the name of the garden bed")
+    description = models.TextField(blank=True, help_text="Enter a brief description of the garden bed")
 
     def __str__(self):
         return self.name
 
+    class Meta:
+        verbose_name = "Garden Bed"
+        verbose_name_plural = "Garden Beds"
+        ordering = ["name"]
+
+    def add_plant(self, plant, location):
```

Comment:
Convenience method to add a plant to a garden bed, which encapsulates creating the `Planting` join model instance.